components/search_hint/search_hint.tsx

```diff
@@ -134,6 +163,50 @@ const SearchHint = (props: Props): JSX.Element => {
                         </div>
                     </li>))}
             </ul>
+            {props.recentSearches && <>
```

Comment:
I think we might need to check the length of `recentSearches` here. When I don't have any recent search terms, the "Recent searches" heading is still appearing
![Screen Shot 2022-10-20 at 2 47 07 PM](https://user-images.githubusercontent.com/3277310/197032430-29c676e2-c0e9-4782-9efb-718ba10490aa.png)

Reply:
Good catch!